0800-appendix-197360.Rmd

```diff
@@ -37,22 +37,25 @@ PSCIS crossing `r as.character(my_site)` is located on `r my_pscis_info() %>% pu
+At the time of reporting in 2020, as an initiative of the Forest and Range Evaluation Program as well as the Bulkley Valley Research Centre, FLNR and the BC Ministry of Environment and Climate Change Strategy were undertaking a watershed status evaluation of the Owen Creek watershed [@pickard_etalDRAFTOwen].  The evaluation uses remotely sensed and field based surveys to interpret the current functioning condition of the watershed as well as its possible future state as a result of continuing human and natural activities by ranking eight indicators of watershed pressure related to riparian health, fish passage and fine sediment delivery [@pickard_etal2014FishValues; @porter_etal2019WatershedStatus]. 
 
 <br>
 
-PSCIS stream crossing `r as.character(my_site)` was prioritized for follow up with a habitat confirmation through consultation with Lars Reese-Hansen and Don Morgan (Wildlife Habitat Specialist, MoE). They indicated (pers comm.) that following fish passage assessments in 2014 (implemented through watershed status evaluation field surveys throughout the Owen Creek watershed), Riddeck Creek upstream of the crossing contained the highest value habitat potentially blocked by a road-stream crossing barrier. A map of the watershed is provided in map attachment [`r my_bcfishpass() %>% pull(dbm_mof_50k_grid)`](`r my_mapsheet()`).
+PSCIS stream crossing `r as.character(my_site)` was prioritized for follow up with a habitat confirmation through consultation with Lars Reese-Hansen and Don Morgan (Wildlife Habitat Specialist, MoE). They indicated (pers comm.) that following the fish passage assessments in 2014 (implemented through watershed status evaluation field surveys throughout the Owen Creek watershed), Riddeck Creek upstream of the crossing contained the highest value habitat potentially blocked by a road-stream crossing barrier. Based of these recommendations, a habitat assessment was conducted during the initial 2020 survey. A map of the watershed is provided in map attachment [`r my_bcfishpass() %>% pull(dbm_mof_50k_grid)`](`r my_mapsheet()`).
 
+<br>
+
+In 2022, planning began to replace crossing `r as.character(my_site)` as part of the Bii Wenii Kwa Restoration/Recovery Plan lead by the Office of the Wet’suwet’en (@pickard_etal2023BiiWenii). As of 2023, engineering plans have been completed, some materials have be purchased, and the replacement cost is estimated at $900,000.  On September 13, 2023, crossing `r as.character(my_site)` was revisited and electrofishing was conducted for baseline monitoring and to scope again for anadromous species. Fish sampling results are included in this report, along with the initial 2020 results. 
```

Comment:
@pickard_etal2023BiiWenii is watershed status assessment vs the restoration plan.  All our work since 2020 is just as relevent as @pickard_etal2023BiiWenii.